Dockerfile

```diff
@@ -1,42 +1,33 @@
-FROM ubuntu:xenial
+FROM ubuntu:eoan
```

Comment:
please consider switching to an LTS version - ie. 18.04 bionic or 20.04 focal (note that focal support will be added in 1-2 weeks: https://swicloud.atlassian.net/browse/AO-16324 )